app/controllers/projects_controller.rb

```diff
@@ -1,5 +1,21 @@
 class ProjectsController < ApplicationController
   def review_turnaround
-    @metrics = Queries::DailyMetrics.call(params[:project_id])
+    return if params.dig(:metric, :period).blank?
+
+    @metrics = if metric_params[:period] == 'weekly'
+                 Queries::WeeklyMetrics.call(project_id)
+               else
+                 Queries::DailyMetrics.call(project_id)
```

Comment:
```suggestion
               elsif metric_params[:period] == 'daily'
                 Queries::DailyMetrics.call(project_id)
               else
                 raise UnknownDateRange
               end
```

minor comment, but maybe it's good to avoid confusion. e.g. if someone sends 'monthly' they'd still be getting daily values